### ISSUE TITLE
Java controllers crash

### ISSUE THREAD

**Issue by omichel** [cyberbotics/webots#2545]
When running the java language example, I am getting the following crash of the Java controller:
```
PointerException
	at com.cyberbotics.webots.controller.Robot.getOrCreateDevice(Robot.java:327)
	at com.cyberbotics.webots.controller.Robot.getCamera(Robot.java:75)
	at Slave.<init>(Slave.java:49)
	at Slave.main(Slave.java:108)
WARNING: 'Slave' controller exited with status: 1.
```

**Comment by omichel** [comment]
This bug was introduced in #2508.

**Comment by omichel** [comment]
Fixed in #2546.
